servers/zonecraft/metadata.json

```diff
@@ -12,15 +11,42 @@
         "darktimes.es",
         "bedwars.es",
         "zonehosting.net",
-        "skywars.es"
+        "skywars.es",
+        "pickaxe.es"
+    ],
+    "primaryAddress": "play.zonecraft.es",
+    "primaryMinecraftVersion": "1.8",
+    "minecraftVersions": [
+        "1.8",
+        "1.9",
```

Comment:
this version isn't supported by lunar client please check the README.MD for what versions can go here.